fusor-ember-cli/app/templates/undercloud-deploy.hbs

```diff
@@ -1,20 +1,21 @@
+{{#unless deployed}}
 <div class="row">
   <div class='col-md-9'>
     <form class="form-horizontal" role="form">
         {{text-f label="Undercloud IP"
-                 value=model.undercloudIP
+                 value=undercloudIP
                  labelSize='deploy-undercloud-param-label'
                  inputSize='deploy-undercloud-param-input'
                  isRequired=true
                  helpText=undercloudIPHelp}}
         {{text-f label="SSH User"
-                 value=model.sshUser
+                 value=sshUser
```

Comment:
purposely changed from `model.sshUser` to `sshUser` (in the controller) so that the model is not saved by the frontend.